source/includes/steps-repo.yaml

```diff
@@ -133,7 +133,22 @@ content: |
   .. code-block:: sh
 
      git remote add upstream https://github.com/<repoOwner>/<remoteRepoRoot>.git
+---
+title: Remove the redirects file
+ref: remove-redirects
+content: |
+
+  Some of our repositories have a ``redirects`` file that redirects browsers permanently to a new location.
+
+  Because non-admin users do not have permission to create or modify redirects, please remove the file.
+
+  In the config directory of your repo, run:
 
+  .. code-block:: sh
+
+     rm redirects
```

Comment:
Most of the previous steps are at the repo level. I'd consider just
`rm config/redirects`

but the lead in text makes this ok